libs/completer/completer_test.go

```diff
@@ -0,0 +1,109 @@
+package completer
```

Comment:
Could you add a test which tests when all ReadDir calls fail?

Reply:
Added!